### ISSUE TITLE
Using [x] parameter impacting symbol size

### ISSUE THREAD

**Issue by lostsatellites** [DWilliames/paddy-sketch-plugin#54]
This is an issue template.

Please fill in the relevant fields – whether this is a **Feature request** or a **Bug report**
Remove any sections that are not relevant to you.

---

#### Expected behavior:
The `x` parameter should not impact the padding for edges it is not applied to.

#### Actual behavior:
Using the `x` to ignore padding on certain edges is causing other padding to not work. For example, I have a button with padding `[10 x 11]` for a button with a 40px height on the symbol page. When using the symbol in layout the height changes to 37px. Using an integer instead of `x` maintains the button height at 40px. I have also tried `[10 x 11 x]` and the same height change happens. You can see in the screenshots that the button appears to be getting cropped on the lower portion.

<img width="525" alt="screen shot 2018-02-28 at 12 19 56 pm" src="https://user-images.githubusercontent.com/1437078/36805161-464e63a2-1c82-11e8-8455-6fa5473949a0.png">
<img width="643" alt="screen shot 2018-02-28 at 12 20 25 pm" src="https://user-images.githubusercontent.com/1437078/36805162-465e59ba-1c82-11e8-98c4-1f3843599292.png">

#### Steps to reproduce:
See above. (Tried to attach Sketch file, but got a "We don't support that file type." error message.

**Comment by mikeozornin** [comment]
>(Tried to attach Sketch file, but got a "We don't support that file type." error message.

You can make a zip from sketch file and attach it.

**Comment by DWilliames** [comment]
Yeah that seems quite weird @lostsatellites. Thanks for reaching out.

I think Paddy is struggling with Symbol support at the moment; I'm going to have to re-think my approach for how it is implemented. 🤔
